lib/octicons_react/src/index.d.ts

```diff
@@ -1,16 +1,21 @@
+// eslint-disable-next-line import/no-namespace
 import * as React from 'react'
 
-import {Icon} from './__generated__/icons'
+// eslint-disable-next-line prettier/prettier
+import { Icon } from './__generated__/icons'
 
 type Size = 'small' | 'medium' | 'large'
 
 export interface OcticonProps {
   'aria-label'?: string
+  'aria-labelledby'?: string
   tabIndex?: number
   children?: React.ReactElement<any>
   className?: string
+  title?: string | React.ReactElement<any>
+  id?: string
   fill?: string
-  icon?: Icon
+  icon?: Icon | React.SVGAttributes<SVGElement>
```

Comment:
Just curious, what is the `React.SVGAttributes` bit used for here in `OcticonProps`?